pkgs/games/infra-arcana/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, fetchFromGitLab, cmake, makeWrapper, graphicsmagick,
+  SDL2, SDL2_image, SDL2_mixer }:
+
+stdenv.mkDerivation rec {
+  name = "infra-arcana-${version}";
+  version = "19.2";
+
+  src = fetchFromGitLab {
+    owner = "martin-tornqvist";
+    repo = "ia";
+    rev = "v${version}";
+    sha256 = "0qgfb4djsmh4dbgdz1ahyhkxm6zzhn7vc9xwbvyyb8dzl8abmig7";
+  };
+
+  # The game shows the compile date on the main menu screen. Nix pretends it's
+  # Jan 1 1970 and so it looks funny. This patch hard-codes the datestring to
+  # the release date (taken from git) to preserve determinism while showing
+  # something useful.
+  patches = [ ./release-date.patch ];
+
+  nativeBuildInputs = [ cmake makeWrapper graphicsmagick ];
+
+  buildInputs = [ SDL2 SDL2_image SDL2_mixer ];
+
+  hardeningDisable = [ "fortify" ];
+
+  postInstall = ''
+    cp -r target/ia/res $out
+    install -D -m 0555 target/ia/ia $out/bin/infra-arcana
+    wrapProgram $out/bin/infra-arcana --run "cd $out"
+
+    i=0; for s in 16 32 48 64 128 256; do
+      iconPath=$out/share/icons/hicolor/$s"x"$s/apps
+      mkdir -p $iconPath
+      gm convert -transparent white ../icon/icon.ico\[$i\] \
+        $iconPath/infra-arcana.png
+      i=$((i+1))
+    done
+
+    mkdir -p $out/share/applications
+    cat > $out/share/applications/infra-arcana.desktop <<EOF
+    [Desktop Entry]
+    Type=Application
+    Name=Infra Arcana
+    Comment=A Roguelike set in the early 20th century
+    Icon=infra-arcana
+    Exec=infra-arcana
+    Terminal=false
+    StartupWMClass=.infra-arcana-wrapped
+    Categories=Game;AdventureGame;
+    EOF
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A Roguelike set in the early 20th century";
+    homepage = https://sites.google.com/site/infraarcana;
```

Comment:
I can't go to this homepage (google sh*t) , and you have to use quoted url to follow new guideline. "https://gitlab.com/martin-tornqvist/ia/" seems a better option for me.

Reply:
I wasn't aware of the quoting guidelines, thanks! I updated it to the GitLab URL; I don't really have strong feelings either way.